src/cmd/manage_nsfs.js

```diff
@@ -180,8 +189,8 @@ async function fetch_existing_bucket_data(target) {
     return data;
 }
 
-async function add_bucket(data) {
-    await manage_nsfs_validations.validate_bucket_args(config_root_backend, accounts_dir_path, data, ACTIONS.ADD);
+async function add_bucket(data, account) {
```

Comment:
You don't use the account argument and didn't pass it in the call, it is redundant.

Reply:
Thanks.